### ISSUE TITLE
L10n and I18n support

### ISSUE THREAD

**Issue by CloneWith** [CloneWith/opsu#70]
Make opsu! support multiple languages. Also part of #30 

- [x] Mark strings
- [x] Handle translation files (Properties)
- [x] Font rendering
- [x] Translation platform (e.g. Weblate)

**Comment by CloneWith** [comment]
This is too annoying. I may put it off priority 💀

**Comment by CloneWith** [comment]
Due to the lack of support of slick2D for i18n, I'm going to implement this feature in the next iteration of opsu!

**Comment by CloneWith** [comment]
After scanning some relevant issues in itdelatrisu/opsu I found something is wrong with font rendering.
As far as I've found out, opsu! use an individual font, Droid Sans Fallback, to render CJK characters. It adds only glyphs used by each string instead of adding all at once.

Also, spaces aren't handled well - they are obviously too long when using properties to translate the original string.

This is a problem obviously worths digging into deeply. I'll try to investigate this later and later.
